ts/Extensions/Annotations/Types/Measure.ts

```diff
@@ -452,10 +453,15 @@ class Measure extends Annotation {
         let selectType = this.options.typeOptions.selectType,
             controlPoint;
 
-        if (selectType === 'x') {
-            options.style.cursor = 'ew-resize';
-        } else if (selectType === 'y') {
-            options.style.cursor = 'ns-resize';
+        if (
+            !defined(this.userOptions.controlPointOptions &&
+                this.userOptions.controlPointOptions.style.cursor)
```

Comment:
The value in `options.style.cursor` can be set in `controlPointOptions` (general, for all control points), or in `contrPoints` array. 

Perhaps it will be better (and easier) to check if the option is already set in the object? Like this:
```js
if (!defined(options.style.cursor)) {
   ...
}
```

Reply:
Since the `cursor` is already defined in the `controlPointOptions`, this condition always fails.

Reply:
Aaah, right, fine then! 👍